src/main/java/io/apimatic/core/ApiCall.java

```diff
@@ -117,20 +138,18 @@ public Builder<ResponseType, ExceptionType> responseHandler(
         }
 
         /**
-         * 
          * @param action endpointConfiguration {@link Consumer}
          * @return {@link ApiCall.Builder}
          */
-        public Builder<ResponseType, ExceptionType> endpointConfiguration(
-                Consumer<EndpointConfiguration.Builder> action) {
+        public Builder<ResponseType, ExceptionType>
+                endpointConfiguration(Consumer<EndpointConfiguration.Builder> action) {
```

Comment:
Question: Did the linter enforce you to take the method name to the next line?

Reply:
No, Formatter moved the method name to the next line.